spec/controllers/tutor_comments_controller_spec.rb

```diff
@@ -0,0 +1,154 @@
+require 'rails_helper'
+
+RSpec.describe TutorCommentsController, type: :controller do
+  describe 'endpoints' do
+    before do
+      sign_in_auth
```

Comment:
if you add `@tutor = create(:tutor)` here. you can get rid of all the `@tutor = create(:tutor)` in each method below.